go-controller/pkg/ovn/libovsdbops/nb_global.go

```diff
@@ -0,0 +1,61 @@
+package libovsdbops
+
+import (
+	"context"
+	"fmt"
+
+	libovsdbclient "github.com/ovn-org/libovsdb/client"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+)
+
+// findNBGlobal returns the NBGlobal table entry
+func findNBGlobal(nbClient libovsdbclient.Client) (*nbdb.NBGlobal, error) {
+	nbGlobal := []nbdb.NBGlobal{}
+	ctx, cancel := context.WithTimeout(context.Background(), types.OVSDBTimeout)
+	defer cancel()
+	err := nbClient.List(ctx, &nbGlobal)
+	if err != nil {
+		return nil, fmt.Errorf("can't find Locial Switches err: %v", err)
+	}
+
+	if len(nbGlobal) == 0 {
+		return nil, libovsdbclient.ErrNotFound
+	}
+	// The nbGlobal table should only have one row
+	if len(nbGlobal) != 1 {
+		return nil, fmt.Errorf("multible NBGlobal rows found")
+	}
+
+	return &nbGlobal[0], nil
+}
+
+// SetNBGlobalOptions mutates the options on the NBGlobal table
```

Comment:
update or mutate?

Reply:
So I think this is update (because I fetched the existing nbGlobal object already... I will need to clarify this function